server/controllers/meeting.js

```diff
@@ -3,20 +3,31 @@ const User = require("../models/User");
 const asyncHandler = require("express-async-handler");
 
 exports.createMeeting = asyncHandler(async (req, res, next) => {
-  const { username, duration } = req.body;
-  const loggedInUser = await User.findOne({ username });
-  Meeting.create({
-    userId: loggedInUser._id,
-    duration: duration,
-  });
-  res.status(200).json({ username, duration });
+  const { name, description, duration, url } = req.body;
+
+  const generatedUrl = `/event/${req.user._id}/${url}`;
+  try {
+    const meetingCreated = await Meeting.create({
+      userId: req.user._id,
+      name,
+      description,
+      duration,
+      url: generatedUrl,
+    });
+    if (meetingCreated) {
+      res.status(200).json({ eventId: meetingCreated._id });
+    }
+  } catch (err) {
+    res.status(400).send(err);
+  }
```

Comment:
Express-async-handler will catch thrown errors by Meeting.create() and send a JSON response.